apps/src/templates/lessonOverview/StudentLessonOverview.jsx

```diff
@@ -0,0 +1,150 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import i18n from '@cdo/locale';
+import {announcementShape} from '@cdo/apps/code-studio/announcementsRedux';
+import Announcements from '../../code-studio/components/progress/Announcements';
+import {connect} from 'react-redux';
+import {SignInState} from '@cdo/apps/templates/currentUserRedux';
+import {ViewType} from '@cdo/apps/code-studio/viewAsRedux';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import InlineMarkdown from '@cdo/apps/templates/InlineMarkdown';
+import styleConstants from '@cdo/apps/styleConstants';
+import color from '@cdo/apps/util/color';
+import LessonNavigationDropdown from '@cdo/apps/templates/lessonOverview/LessonNavigationDropdown';
+import {studentLessonShape} from '@cdo/apps/templates/lessonOverview/lessonPlanShapes';
+
+const styles = {
+  header: {
+    margin: '10px 0px',
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  navLink: {
+    fontSize: 14,
+    lineHeight: '22px',
+    color: color.purple,
+    margin: '10px 0px'
+  },
+  titleNoTopMargin: {
+    marginTop: 0
+  }
+};
+
+class StudentLessonOverview extends Component {
+  static propTypes = {
+    lesson: studentLessonShape.isRequired,
+
+    // from redux
+    announcements: PropTypes.arrayOf(announcementShape),
+    isSignedIn: PropTypes.bool.isRequired
+  };
+
+  linkWithQueryParams = link => {
+    const queryParams = window.location.search || '';
+    return link + queryParams;
+  };
+
+  normalizeUrl = url => {
```

Comment:
I think it might be worth it to extract Resources display into a separate component as there is a lot of duplicate code here. I don't want to block this PR on this so I'm happy to own this as a follow-up.

Reply:
I agree it feels a little silly to duplicate this and the vocab section across teacher and student lesson plans. I don't think I have the bandwidth to do that this week so making that a follow up would be great.